edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-batching.adoc

```diff
@@ -0,0 +1,60 @@
+= EDIFACT EDI Connector 2.3 - Batch Outbound EDIFACT Messages - Mule 4
+
+For EDI messages that are not time-sensitive, such as invoice and payment remittances, you can use the Batch Write operation to merge multiple individual messages into a single interchange to send to your trading partners.
+
+== Batching Stages
+
+Use the Anypoint EDIFACT EDI Connector Batch Write operation to merge individual EDIFACT messages into a single transmission to send to your trading partners.
+
+Implementing a batch involves two stages:
+
+. Accumulate the transactions.
+. Release the batch.
+
+=== Accumulate the Transactions
+
+To accumulate transactions for a batch:
+
+. Transform the payloads into EDIFACT messages using DataWeave and EDIFACT as you receive application messages (such as a JSON invoice) from the backend systems.
+. Create EDIFACT payloads using temporary control number keys in the EDIFACT Write operation. +
+This enables sending the accumulated EDIFACT as a batch later. The batch is created using control numbers that are not used in the final batch sent to the trading partner, for example, `<edifact:write doc:name="Write individual D97A ORDERS" doc:id="3d51215a-125d-461b-9791-0a939e5ebc45" config-ref="EDIFACT_Write_ORDERS" interchangeControlNumberKey="NTO-MYTHICAL-UNB-TMP"/>`.
+. Store the intermediate EDIFACT message payloads in your choice of storage, such as a database table:
+
+image::edifact-edi-connector-batch-flow-1.jpg[Accumulate the transactions]
+
+For example, this implementation of a Mule flow:
+
+* Receives JSON payloads from the backend systems.
+* Transforms to an `EDIFACT D97A ORDERS` message.
+* Generates the intermediate EDIFACT message payload using the EDIFACT Write operation.
+* Stores the EDIFACT payload in a database table.
+
+=== Release the Batch
+
+Trading partners have varying criteria for how they want transactions batched:
+
+* Release the batch at a specific time interval or a specific time of the day.
+* Release the batch when a specific number of transactions is accumulated.
```

Comment:
accumulated > reached (or similar)
"a number" is not "accumulated," although "transactions" are...in which the verb becomes plural: "are accumulated."